tests/test_30_decompressors.py

```diff
@@ -327,10 +277,11 @@ def test_40_7z_multiple_files(self):
         )
         raw.name = "test_file.7z"
         self._check_decompressor(
-            destream.decompressors.Un7z, raw, uncompressed
+            tmp_path, destream.decompressors.Un7z, raw, uncompressed
         )
 
-    def test_30_zip_single_file(self):
+    @pytest.mark.xfail(reason="libmagic 5.39 reports wrong mime-type")
+    def test_30_zip_single_file(self, tmp_path):
```

Comment:
```suggestion
    def test_30_zip_single_file(self, tmp_path):
```

> test_30_zip_single_file fails for file/libmagic 5.39, marked it as xfail for now, discussed in #20

What? You can't disable an entire test because something fails on a specific version. Imagine someone would work on the zip thingy and really break it and it goes to production?

What you can do is:
1. Make sure the CI is not using the faulty version so the test runs and succeed
2. Make a code that would read the version of libmagic and skip the test
3. Don't forget to add a note on that in the "troubleshooting" section of the README

```
def test_function():
    if not check_lib_magic_version():
        pytest.skip("unsupported libmagic version")
```

Reply:
(xfail: that's why it is in draft, I wanted to discuss it all with you first)

Reply:
Oh ok sorry :sweat_smile: let me know if you struggle on one and I will give it a try. I don't want to leave anything broken :slightly_smiling_face: